pallets/loans/src/mock.rs

```diff
@@ -135,6 +138,33 @@ impl pallet_balances::Config for Runtime {
     type WeightInfo = pallet_balances::weights::SubstrateWeight<Runtime>;
 }
 
+pub type TimeStampedPrice = orml_oracle::TimestampedValue<OraclePrice, Moment>;
+pub struct MockDataProvider;
+impl DataProvider<CurrencyId, TimeStampedPrice> for MockDataProvider {
+    fn get(currency_id: &CurrencyId) -> Option<TimeStampedPrice> {
+        match *currency_id {
+            DOT => Some(TimeStampedPrice {
+                value: OraclePrice::saturating_from_integer(100),
+                timestamp: 0,
+            }),
+            _ => None,
+        }
+    }
+}
+
+pub struct OneToFive;
+impl SortedMembers<AccountId> for OneToFive {
+    fn sorted_members() -> Vec<AccountId> {
+        vec![1, 2, 3, 4, 5]
+    }
+}
+
+impl pallet_prices::Config for Runtime {
+    type Event = Event;
+    type Source = MockDataProvider;
+    type FeederOrigin = EnsureSignedBy<OneToFive, AccountId>;
```

Comment:
This is overkill, how about only signed by 1 account?
https://github.com/paritytech/substrate/blob/master/frame/node-authorization/src/mock.rs#L76